cmd/gctcli/data_history.go

```diff
@@ -0,0 +1,505 @@
+package main
```

Comment:
Since one of my next PRs will continue on data integrity, I'll look into pausing jobs there and candle conversions there 👍 Pausing all jobs however sounds like disabling the subsystem.